plenum/test/txn_author_agreement/conftest.py

```diff
@@ -27,3 +33,23 @@ def config_req_handler(config_state,
     return ConfigReqHandler(config_ledger,
                             config_state,
                             domain_state=FakeSomething())
+
+
+@pytest.fixture
+def taa_in_data():
+    return [
+        TaaData(randomString(8), randomString(32), n, n + 10)
+        for n in range(10)
+    ]
+
+
+@pytest.fixture
+def taa_state_data(taa_in_data):
+    return {data.version: expected_state_data(data) for data in taa_in_data}
+
+
+@pytest.fixture
+def taa_digests(taa_in_data):
```

Comment:
After reading through tests I think this is better be named like `taa_expected_digests`

Reply:
updated